fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/insert/OlapInsertExecutor.java

```diff
@@ -147,13 +151,15 @@ public void finalizeSink(PlanFragment fragment, DataSink sink, PhysicalSink phys
         } catch (Exception e) {
             throw new AnalysisException(e.getMessage(), e);
         }
-        TransactionState state = Env.getCurrentGlobalTransactionMgr().getTransactionState(database.getId(), txnId);
-        if (state == null) {
-            throw new AnalysisException("txn does not exist: " + txnId);
-        }
-        addTableIndexes(state);
-        if (physicalOlapTableSink.isPartialUpdate()) {
-            state.setSchemaForPartialUpdate((OlapTable) table);
+        if (!isGroupCommitHttpStream()) {
```

Comment:
Try the following pattern to reduce indention and possible conflict
```suggestion
        if (isGroupCommitHttpStream()) {
            return;
        }
```